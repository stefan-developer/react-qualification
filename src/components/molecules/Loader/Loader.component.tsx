import React from "react";
import * as Styled from "./Loader.styles";

function Loader(): JSX.Element {
  return (
    <Styled.Wrapper>
      <p>Loading ...</p>
    </Styled.Wrapper>
  );
}

export default Loader as React.FC;
