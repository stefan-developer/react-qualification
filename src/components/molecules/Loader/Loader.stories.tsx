import React from "react";

import { storiesOf } from "@storybook/react";

import Loader from "./Loader.component";

storiesOf("Loader", module).add("default", () => <Loader />);
