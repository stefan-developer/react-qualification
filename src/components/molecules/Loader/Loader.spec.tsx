import React from "react";
import renderer from "react-test-renderer";

import Loader from "./Loader.component";

test("Loader renders as expected", () => {
  const component = renderer.create(<Loader />);

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
