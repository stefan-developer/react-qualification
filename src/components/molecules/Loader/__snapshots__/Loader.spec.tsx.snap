// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders as expected 1`] = `
<div
  className="sc-AxjAm fDsnWW"
>
  <p>
    Loading ...
  </p>
</div>
`;

exports[`Loader renders as expectede 1`] = `
<div
  className="sc-AxjAm fDsnWW"
>
  <p>
    Loading ...
  </p>
</div>
`;
